runtime/gc_vlhgc/WriteOnceCompactor.cpp

```diff
@@ -1651,6 +1663,12 @@ class MM_WriteOnceCompactFixupRoots : public MM_RootScanner {
 		scanJVMTIObjectTagTables(env);
 #endif /* J9VM_OPT_JVMTI */
 
+#if defined(J9VM_GC_SPARSE_HEAP_ALLOCATION)
+	if (_includeVirtualLargeObjectHeap) {
+		scanObjectsInVirtualLargeObjectHeap(env);
```

Comment:
why we don't override doObjectInVirtualLargeObjectHeap for WOC? seems like we should update offheap (dataAddr->spine) mapping when spine moved